StatusList.js

```diff
@@ -1,9 +1,9 @@
 /*!
- * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
  */
 import Bitstring from '@digitalbazaar/bitstring';
 
-export default class RevocationList {
+export default class StatusList {
```

Comment:
If this is a `StatusList`, why does it have `setRevoked` and `isRevoked` in it? Isn't this just a `RevocationList` again? Should there be a base class that does `setStatus` -- and `RevocationList` inherits and calls that internally from `setRevoked`? Or would the base class just suffice -- and you need to know what you're working with?

Something here needs adjustment.

Reply:
I have updated RevocationList to be a base class here [7d1a7b6](https://github.com/digitalbazaar/vc-status-list/pull/1/commits/7d1a7b6b833805037cd74e1a4fa1a6a4e1b7adff) Is this what you had in mind?